counter-sink/src/main/resources/application.yml

```diff
@@ -5,4 +5,4 @@ spring:
   cloud:
     stream:
       bindings:
-        input: counter
+        input: ${remote.name:application:${server.port}}
```

Comment:
Actually, `server.port` should be left at its default value (8080) unless the developer of a module has another reason to change that.